website/docs/FAQ.md

```diff
@@ -22,7 +22,7 @@ assistant = autogen.AssistantAgent(
 )
 ```
 
-The `llm_config` is used in the [`create`](/docs/reference/oai/client#create) function for LLM inference.
+The `llm_config` is used in the [`create`] <!-- (/docs/reference/oai/client#create)--> function for LLM inference.
```

Comment:
Why is the link commented out?

Reply:
The build fails because the files doesn't exist

Reply:
@olaoluwasalami I think you should figure out the new path to this file. Probably has autogen in it or something like that.